# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/base/CacheConstants.h
# Commit: f6ce0e05c92e
# Full Hash: f6ce0e05c92e16e33c4bc136228ab4bc6058f358
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2022-01-05 09:33:53
# Regressor Bug: 1737944
# File Overlap Count: 2
# Description:
#   Bug 1737944 - P4: Cache aria group attributes and introduce Accessible getter. r=Jamie
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D134207
# ==============================================================================

diff -r afea577be988 -r f6ce0e05c92e accessible/base/CacheConstants.h
--- a/accessible/base/CacheConstants.h	Tue Jan 04 21:01:35 2022 +0000
+++ b/accessible/base/CacheConstants.h	Tue Jan 04 21:01:35 2022 +0000
@@ -19,6 +19,7 @@
   static constexpr uint64_t Text = ((uint64_t)0x1) << 4;
   static constexpr uint64_t DOMNodeID = ((uint64_t)0x1) << 5;
   static constexpr uint64_t State = ((uint64_t)0x1) << 6;
+  static constexpr uint64_t GroupInfo = ((uint64_t)0x1) << 7;
   static constexpr uint64_t All = ~((uint64_t)0x0);
 };
 